src/diffusers/pipelines/lavender_flow/pipeline_lavender_flow.py

```diff
@@ -448,20 +450,15 @@ def __call__(
 
         # 6. Denoising loop
         num_warmup_steps = max(len(timesteps) - num_inference_steps * self.scheduler.order, 0)
-        dt = 1.0 / num_inference_steps
-        dt = (
-            torch.tensor([dt] * effective_batch_size)
-            .to(self.device)
-            .view([effective_batch_size, *([1] * len(latents.shape[1:]))])
-        )
         with self.progress_bar(total=num_inference_steps) as progress_bar:
-            for i, t in enumerate(range(num_inference_steps, 0, -1)):
+            for i, t in enumerate(timesteps):
                 # expand the latents if we are doing classifier free guidance
                 latent_model_input = torch.cat([latents] * 2) if do_classifier_free_guidance else latents
                 # broadcast to batch dimension in a way that's compatible with ONNX/Core ML
-                t = t / num_inference_steps
                 timestep = (
-                    torch.tensor([t]).expand(latent_model_input.shape[0]).to(latents.device, dtype=latents.dtype)
+                    torch.tensor([t / 1000])
+                    .expand(latent_model_input.shape[0])
+                    .to(latents.device, dtype=latents.dtype)
```

Comment:
We could unpack this code a bit and perhaps add a comment on why we're dividing by 1000. This is mainly because we're moving away from our normal pipeline implementations a bit which our users are used to reading and referring to.

Reply:
done! feel free to merge into your PR now